clarity-ext-scripts/clarity_ext_scripts/covid/parse_pcr_execution.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+import pandas as pd
+from datetime import datetime
+
+CT_HEADER = u"Cт"
+
+
+class ParsePcrExecution(object):
+    def __init__(self, context):
+        self.context = context
+
+    def execute(self):
+        file_name = "Result file"
+        f = self.context.local_shared_file(file_name, mode="rb")
+        data = pd.read_excel(f, 'Results', index_col=0, skiprows=7, encoding='utf-8')
+        rt_pcr_analyze_service = RTPCRAnalyseService(self.context)
+
+        for _, output in self.context.all_analytes:
+            entry = data.loc[output.well.alpha_num_key]
+            target_name = entry["Target Name"]
+
+            if target_name != output.name:
+                raise AssertionError("Incorrect name of target '{}' in well '{}' in '{}'. "
+                        "Expected {}".format(
+                            target_name, output.well.alpha_num_key, file_name, output.name))
+
+            ct = entry[CT_HEADER]
```

Comment:
I don't understand how this works, since we do need to parse a FAM and a HEX/VIC entry for each sample. From what I understand here there only a single CT value is stored per analyte here. Am I misunderstanding something.